src/issue-strategies/bug-maintenance.js

```diff
@@ -1,7 +1,16 @@
+import {findParent} from '../jira-operations';
+
 export function apply(issue, jiraClientAPI) {
   if(issue === null || issue.fields.status.statusCategory.colorName !== 'yellow') {
     return Promise.reject(new Error(`Issue ${issue.key} is not open to commit against`));
   }
-
-  return Promise.resolve(true);
+  
+  if(!issue.fields.issuelinks)
+  {
+    return Promise.resolve(true);
+  }
+  
+  return findParent(issue, jiraClientAPI).then((parent)=>
+      (!parent || parent.fields.status.statusCategory.colorName === 'yellow') ? Promise.resolve(true) : Promise.reject(new Error(`Cannot commit issue ${issue.key} because parent issue ${parent.key} is not available to commit against.`))
+    );
```

Comment:
I thought we could get rid of that `issuelinks` check. What happens when we invoke `findParent` when `issuelinks` is falsy?